src/AutoRest.CSharp.V3/Generation/Writers/CodeWriter.cs

```diff
@@ -118,7 +119,14 @@ public CodeWriter Append(FormattableString formattableString)
                     default:
                         if (isLiteral)
                         {
-                            Literal(argument);
+                            if (argument is EnumTypeValue enumTypeValue)
```

Comment:
Let's move this to WriteConstant method. It does a lot of runtime constant writing there.